hivemind_bus_client/http.py

```
@@ -0,0 +1,321 @@
+
+    def _handle_binary(self, message: HiveMessage):
+        assert message.msg_type == HiveMessageType.BINARY
+        bin_data = message.payload
+        LOG.debug(f"Got binary data of type: {message.bin_type}")
+        if message.bin_type == HiveMindBinaryPayloadType.TTS_AUDIO:
+            lang = message.metadata.get("lang")
+            utt = message.metadata.get("utterance")
+            file_name = message.metadata.get("file_name")
+            try:
+                self.bin_callbacks.handle_receive_tts(bin_data, utt, lang, file_name)
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)